nikola/nikola.py

```diff
@@ -455,6 +455,7 @@ def __init__(self, **config):
             'FRONT_INDEX_HEADER': '',
             'GALLERY_FOLDERS': {'galleries': 'galleries'},
             'GALLERY_SORT_BY_DATE': True,
+            'GALLERY_USE_METADATA': True,
```

Comment:
I don't think we need an option, since it's unlikely that metadata files will randomly appear next to images. If people don't want to show image metadata, then don't put meta files there :-)